BackofficeBundle/Resources/public/less/editTable.less

```diff
@@ -1,7 +1,54 @@
-.back-to-list {
-    margin-bottom: 20px;
+#widget-grid .submit_form,
```

Comment:
You should use the less syntax that encapsulate children in their parent. It's one of the best feature of less :

    #widget-grid {
        .submit_form, .submit_form:active, .back-to-list, [...] {
            width: 119px;
        }
        .back-to-list {
            position: fixed;
            top: 54px;
            [....]
        }
    }